contracts/IColony.sol

```diff
@@ -407,4 +408,9 @@ contract IColony {
   /// @param _token Address of the token, `0x0` value indicates Ether
   /// @return Total amount of tokens in pots other than the rewards pot (id 0)
   function getNonRewardPotsTotal(address _token) public view returns (uint256);
+
+  /// @notice Convert old- and new-style ratings to new-style ratings
+  /// @param rating Original rating
+  /// @return New-style rating
+  function convertRating(uint8 rating) public pure returns(uint8);
```

Comment:
Does this function need to be public and/or on the interface?

Reply:
I made it public so I could test it; otherwise the code coverage build would fail.